### PR TITLE
Secure/fix URLs

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/byobu.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Byobu < Formula
   desc "Text-based window manager and terminal multiplexer"
-  homepage "http://byobu.co/"
+  homepage "https://github.com/dustinkirkland/byobu"
```
Comment (fxcoudert) [inline Formula/byobu.rb]: github repo is a mirror, not the official upstream

File: Formula/libggz.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Libggz < Formula
   desc "The libggz library wraps many common low-level functions"
-  homepage "http://dev.ggzgamingzone.org/libraries/libggz/"
+  homepage "https://web.archive.org/web/20180511124057/dev.ggzgamingzone.org/libraries/libggz/"
```
Comment (fxcoudert) [inline Formula/libggz.rb]: This software is abandoned, not maintained, it has 2 installs in 90 days, let's remove the formula.

File: Formula/libguess.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 class Libguess < Formula
   desc "Character set guessing library"
-  homepage "http://atheme.org/projects/libguess.html"
+  homepage "https://github.com/kaniini/libguess"
```
Comment (fxcoudert) [inline Formula/libguess.rb]: This software is abandoned, not maintained, it has 2 installs in 90 days, let's remove the formula.

File: Formula/lysp.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 class Lysp < Formula
   desc "Small Lisp interpreter"
-  homepage "http://www.piumarta.com/software/lysp/"
-  url "http://www.piumarta.com/software/lysp/lysp-1.1.tar.gz"
+  homepage "https://web.archive.org/web/20181230044600/www.piumarta.com/software/lysp/"
+  url "https://web.archive.org/web/20160503234302/www.piumarta.com/software/lysp/lysp-1.1.tar.gz"
```
Comment (fxcoudert) [inline Formula/lysp.rb]: Unmaintained, requires heavy patching (and non system compiler) to build. Let's remove it from core.

File: Formula/ne.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 class Ne < Formula
   desc "The nice editor"
-  homepage "http://ne.di.unimi.it/"
+  homepage "https://web.archive.org/web/20190402114137/ne.di.unimi.it/"
```
Comment (fxcoudert) [inline Formula/ne.rb]: Let's use the github page instead: https://github.com/vigna/ne

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/byobu.rb (hunk 1 above)
Comment (vszakats) [reply Formula/byobu.rb]: The repository is listed on the original page:
https://web.archive.org/web/20190509202352/byobu.co/source.html
Screenshot 2019-07-20 at 20 20 51

Comment (fxcoudert) [reply Formula/byobu.rb]: But the repo clearly states:

> git mirror of upstream bzr at http://launchpad.net/byobu

and the launchpad page has a lot more info, so it makes sense for us to use it

Comment (vszakats) [reply Formula/byobu.rb]: Okay!

**Comment by vszakats** [comment]
@fxcoudert Thanks for the review/suggestions — things should be fixed now.

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**